src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -81,17 +84,135 @@
                     <RowDefinition Height="*" />
                 </Grid.RowDefinitions>
 
-                <TabControl TabStripPlacement="Left" Background="{StaticResource PreferencesWindowBackgroundColor}">
+                <TabControl TabStripPlacement="Left" 
+                            Background="{StaticResource PreferencesWindowBackgroundColor}">
+                    
+                    <!--General Tab-->
                     <TabItem Header="{x:Static p:Resources.PreferencesViewGeneralTab}"
                              Style="{StaticResource LeftTab}">
-                        <Label Content="General Content goes here..." 
-                               Foreground="{StaticResource PreferencesWindowFontColor}"/>
+
+                        <!--This Grid contains controls for selecting the Language-->
+                        <Grid x:Name="GeneralTab" 
+                              Margin="12 0 0 0"
+                              HorizontalAlignment="Left">
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="*" />
+                            </Grid.RowDefinitions>
+                            <Grid Grid.Row="0">
+                                <StackPanel Orientation="Vertical">
+                                    <Label  Content="Language" 
+                                            Grid.Row="0"
+                                            Foreground="{StaticResource PreferencesWindowFontColor}"/>
+                                    <ComboBox Grid.Row="1" 
+                                              Width="213"
+                                              HorizontalAlignment="Left"
+                                              ItemsSource="{Binding Path=LanguagesList}"
+                                              SelectedItem="{Binding Path=SelectedLanguage}"
+                                              Style="{StaticResource NoBordersComboBoxStyle}">
+                                    </ComboBox>
+                                </StackPanel>
+                            </Grid>
+
+                            <!--This Grid contains controls for selecting Run Settings options-->
+                            <Grid Grid.Row="1">
+                                <Grid.RowDefinitions>
+                                    <RowDefinition Height="Auto" />
+                                    <RowDefinition Height="*" />
+                                </Grid.RowDefinitions>
+                                <Label Grid.Row="0"
+                                   Content="Default Run Settings" 
```

Comment:
Same here

Reply:
@QilongTang 
I searched in the resources files but we don't have a similar entry, then I added a new resource entry, see the changes in the commit: https://github.com/DynamoDS/Dynamo/pull/11542/commits/9035f3d2b7de82a15cb6098d2a56a00c1b06836d